test/integration/targets/old_style_vars_plugins/deprecation_warning/vars.py

```diff
@@ -2,7 +2,7 @@
 
 
 class VarsModule(BaseVarsPlugin):
-    REQUIRES_WHITELIST = False
+    REQUIRES_WHITELIST = True
```

Comment:
I thought we renamed this?

Reply:
This is testing the deprecation warning so I was going to delete it once the code can be deleted. We have a test for REQUIRES_ENABLED.